WORKSPACE

```diff
@@ -304,5 +306,72 @@ rbe_autoconfig(
 
 load("@build_bazel_integration_testing//tools:repositories.bzl", "bazel_binaries")
```

Comment:
do we still use this?

Reply:
yes. we're using the `bazel_binaries()`:
```
BAZEL_VERSION = "0.28.1"
BAZEL_BINARY = "@build_bazel_bazel_%s//:bazel_binary" % BAZEL_VERSION.replace(".", "_")
```
but is only a small portion of `@build_bazel_integration_testing` which we could either vendor in or reproduce and improve on. It doesn't do much except download and extract a bazel release in a repository rule and then setup a filegroup with the extracted files.

The `test_runner.js` has to make assumptions about the name of the bazel executable:
```
const bazelBinary =
    require.resolve(`${config.bazelBinaryWorkspace}/bazel${isWindows ? '.exe' : ''}`);
```
which could be improved on.